README.md

```diff
@@ -46,6 +46,15 @@ Since the project is actively being developed, with new features added regularly
 pip install git+git://github.com/cytomining/pycytominer@2aa8638d7e505ab510f1d5282098dd59bb2cb470
 ```
 
+If you want to perform data collation inside pycytominer, you will need an extra package; this will change your installation commands slightly
```

Comment:
Do you mean in terms of a conda package recipe for pycytominer (which I don't know if you have or not), or in terms of "adding pycytominer to an existing conda environment"?

The TLDR is that [conda doesn't support this or anything like it](https://github.com/conda/conda/issues/7502), though folks have been asking for years. If adding to a conda environment and wanting to use collate (or get all the tests to pass!), one will need to add both pycytominer and `cytominer-database==0.3.4` to the conda environment. In terms of making a package recipe, the linked PR has some ideas on how to do it.

Reply:
I mean through a conda install ([pycytominer is on conda](https://anaconda.org/conda-forge/pycytominer)).

>  If adding to a conda environment and wanting to use collate (or get all the tests to pass!), one will need to add both pycytominer and cytominer-database==0.3.4 to the conda environment.

Gotcha! Can you add some version of this detail in your PR?

Reply:
>Can you add some version of this detail in your PR?

Yup, will do.

>([pycytominer is on conda](https://anaconda.org/conda-forge/pycytominer))

Ok, but how is it being built there, is it a recipe you made, someone else made, etc? Because once this is pulled there essentially there will be two different "flavors" of pycytominer and the conda recipe will presumably need to be updated.

Reply:
Yep, we maintain the [pycytominer conda recipe](https://github.com/conda-forge/pycytominer-feedstock#feedstock-maintainers) and control releases there too

Reply:
Ok; I think something like [this](https://github.com/conda/conda/issues/7502#issuecomment-415182396) is likely the way to go, lmk if you want me to do it/want my help as you get ready to tag this in a future release.

Reply:
Perfect. Yes! We'd love the help.

I added relevant documentation to make sure this stays on radar